experiment/scheduler.py

```diff
@@ -354,15 +356,41 @@ def _get_preempted_replacements(self,
             if self.can_start_preemptible():
                 # See if we can replace with a preemptible.
                 replacements.append(replace_trial(trial, preemptible=True))
+
+                self.num_converted_trials += 1
+                logs.info(
+                    'Restarting a preemptible trial as a preemptible one: '
+                    'Benchmark-Fuzzer pair: %s-%s.'
+                    'Trial ID %d, restarted %3.2f%% trials (%d/%d) as '
+                    'preemptilbes so far.', trial.benchmark, trial.fuzzer,
+                    trial, self.num_converted_trials / self.num_trials * 100,
+                    self.num_converted_trials, self.num_trials)
                 continue
 
             if self.can_start_nonpreemptible(nonpreemptible_starts):
                 # If a trial can't be replaced with a preemptible see if we can
                 # replace it with a nonpreemptible.
                 nonpreemptible_starts += 1
                 replacements.append(replace_trial(trial, preemptible=False))
+
+                logs.info(
+                    'Restarting a preemptible trial as a nonpreemptible one: '
+                    'Benchmark-Fuzzer pair: %s-%s.'
+                    'Trial ID %d, restarted %3.2f%% trials (%d/%d) as '
+                    'nonpreemptibles so far.', trial.benchmark, trial.fuzzer,
+                    trial, nonpreemptible_starts / self.num_trials * 100,
+                    nonpreemptible_starts, self.num_trials)
```

Comment:
nit: both this and the log above are very similar. Can we create a helper function and pass in only the parameters that change?